css-transitions-2/Overview.bs

```diff
@@ -120,6 +122,42 @@ the reversing shortening factor.
 The 'transition-delay' property specifies the [=start delay=]
 of the transition's associated [=animation effect=].
 
+## The 'transition-behavior' Property ## {#transition-behavior-property}
+
+The 'transition-behavior' property specifies whether transitions will be
+started or not for discrete properties.
+
+<pre class="propdef">
+  Name: transition-behavior
+  Value: normal | allow-discrete
```

Comment:
This needs to allow a list of values, not just a single value.  To do that you should probably make this `<transition-behavior-value>#`.  See https://drafts.csswg.org/css-backgrounds-3/#background-attachment for an example of what this should look like (and probably example bikeshed syntax too).

Reply:
done, thanks!